### ISSUE TITLE
https://cppcli.net/ failes in Firefox

### ISSUE THREAD

**Issue by Axel-Naumann** [SdtElectronics/cline#5]
Hi,

`console.js` throws an uncaught exception in `console.js:170`:
```JavaScript
recoverInput() {
        this.input.contentEditable = "plaintext-only";
```
```
SyntaxError: An invalid or illegal string was specified
```
See https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/contenteditable

Thanks for this super cool tool!

**Comment by SdtElectronics** [comment]
I just did a hot fix and hopefully it works now. I can't recall for what reason I opted to `plaintext-only` instead of `true`, so this might cause other problems, therefore I won't commit this change immediately. I'll keep this issue open until the change is committed.

**Comment by SdtElectronics** [comment]
Fixed in [91d7790](https://github.com/SdtElectronics/cline/commit/91d77906f898ffd6f855465e36c05cb49be97a49)
